src/components/ShippingDetails/__tests__/ShippingDetails.test.js

```diff
@@ -3,14 +3,32 @@ import { create } from 'react-test-renderer';
 
 import ShippingDetails from '../ShippingDetails';
 
-jest.mock('react-router-dom', () => ({
-  Link: 'MockedLink'
-}));
+jest.mock('../../Icons/Shipping', () => 'Shipping');
 
 describe('COMPONENT - ShippingDetails', () => {
-  it('renders ShippingDetails component', () => {
-    const component = create(<ShippingDetails />);
+  it('Renders correctly ShippingDetails component', () => {
+    const props = {
+      minDate: 'Tue. 02 19',
+      maxDate: 'Wed. 03. 19',
+      shippingPromo: 'Free after $85'
+    };
+
+    const component = create(<ShippingDetails {...props} />);
 
     expect(component.toJSON()).toMatchSnapshot();
   });
+
+  describe('When there is no shippingPromo', () => {
+    it('Renders correctly ShippingDetails component without shipping promo', () => {
+      const props = {
+        minDate: 'Tue. 02 19',
+        maxDate: 'Wed. 03. 19',
+        shippingPromo: null
```

Comment:
Rather omit that field than pass null (overall avoid _null_ if possible). So just:
```
const props = {
  minDate: ...,
  maxDate: ...
}
```